indigo/distribution.yaml

```diff
@@ -774,6 +774,17 @@ repositories:
       url: https://github.com/ros/bond_core.git
       version: master
     status: maintained
+  bowpmap_ros:
+    release:
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/kejriwalnishant1990/bowpmap_ros.git
+      version: 1.0
+    source:
+      type: git
+      url: https://github.com/kejriwalnishant1990/bowpmap_ros.git
+      version: indigo
```

Comment:
There is no `indigo` branch on this repository. The only current branch is `master`